README.md

```diff
@@ -6,16 +6,133 @@
 
 # Homebase Lite
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT) [![Build status](https://badge.buildkite.com/680d303398dd927c109b9c0324e7eacf9c7e2e422210ffb746.svg?branch=master)](https://buildkite.com/serokell/homebase-lite)
 
 Homebase Lite is a Tezos smart contract for a decentralized voting system, based
 on “snapshots” of the blockchain.
 
 The contract specification is available in [`docs/specification.md`](docs/specification.md)
 
-## Build Instructions
+## Prerequisites
 
-Run `make` to build everything.
+### Build tools
+
+If you want to build this software from source you will need
+[Haskell Tool Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
+
+### Operating system
+
+Please note that only Linux is officially supported.
+If you want to use `tzbtc-client` on MacOS, you can try [building it from source](#build-instructions) using `stack`.
+
+### Executables
+
+In order to use `homebase-lite` executable or run network tests, you will need to obtain `tezos-client` executable.
+`tezos-client` is used for key storing and operation signing.
+
+Various forms of distribution for `tezos-client` are presented in [tezos-packaging repo](https://github.com/serokell/tezos-packaging).
+
+## Obtaining the binary
+
+You can obtain `homebase-lite` binary either by building from source, or downloading an automatically built static executable from [the releases page](https://github.com/tezos-commons/homebase-lite/releases).
+
+### Build instructions
+
+This project is using [Haskell Tool Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/), please follow the link to find instructions on obtaining the `stack` executable.
+
+Once you have `stack`, building should be as easy as cloning the repository and running `stack install` in the root of the cloned tree:
+
+```bash
+$ git clone https://github.com/tezos-commons/homebase-lite.git
+$ cd homebase-lite
+$ stack install
+```
+
+## Using the binary
+
+Once you've obtained `homebase-lite` executable,
+use `homebase-lite --help` to get a list of available commands.
+
+You can also get help for a specific command by running `homebase-lite COMMAND --help`.
+
+### Getting the Michelson code of the compiled contract
+
+Use `homebase-lite print` to get the Michelson code of the contract, e.g.
+
+```bash
+$ homebase-lite print
+```
+
+This will save the contract code to `homebase-lite.tz` in the current working directory. Alternatively, add `--output FILEPATH` key to change the output path, or use `-` to print to standard output:
```

Comment:
Is "key" commonly used or considered to be a more proper term? :thinking: 

I mostly see these referred to as arguments, I think.

Reply:
Hmm. Good point. That's likely a double mistranslation. The proper term here, from which "key" seems to be ultimately derived, is "keyword argument" probably. Other possibilities include "CLI option" or "named argument". The reason I didn't just go with "argument" is because that's colloquially used for positional arguments, at least in my experience, and, well, these aren't positional. What do you reckon?

Reply:
Ah, I see :thinking: 

Well, I think we can take a page out of the [`optparse-applicative` book](https://hackage.haskell.org/package/optparse-applicative-0.17.0.0/docs/Options-Applicative.html#g:3), whose terminology seems reasonable to me and loosely based on the POSIX standards:
> - Flags: simple no-argument options
> - Options: options with an argument
> - Arguments: positional arguments
> - Commands. A command defines a completely independent sub-parser

and simply refer to these as "options".

>  "argument" is because that's colloquially used for positional arguments, at least in my experience

Good point, come to think of it I'd say the same (and it also agrees with the above).